### DISCUSSION TITLE
Populate quickfix(Trouble) list with eslint errors

### DISCUSSION THREAD

**Discussion by skoch13** [LazyVim/LazyVim#1561] [Q&A]
I'm trying to utilize the power of the quickfix list for project-scoped ESLint runs, but currently it's not working as expected.

I set `vim.o.makeprg = "npm run eslint -- --fix --quiet"` and executing it with `make` outputs the result to `noise` opening the linting results as text.

Am I missing something, or is there a completely different approach I should take?

**Comment by guillaumeLamanda** [comment]
You might need the `--format unix` option
